app-catalog/src/components/charts/List.tsx

```diff
@@ -39,53 +46,57 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
     { title: 'Storage', value: 7 },
     { title: 'Streaming and messaging', value: 8 },
   ];
-  const [charts, setCharts] = useState<any[] | null>(null);
+  const [charts, setCharts] = useState<any | null>(null);
   const [openEditor, setEditorOpen] = useState<boolean>(false);
   const [page, setPage] = useState(1);
   const [chartsTotalCount, setChartsTotalCount] = useState(0);
   const [chartCategory, setChartCategory] = useState(helmChartCategoryList[0]);
   const [search, setSearch] = useState('');
   const [selectedChartForInstall, setSelectedChartForInstall] = useState<any | null>(null);
 
+  // note: since we default to true for showVerified and the settings page is not accessible from anywhere else but the list comp
+  // we must have the default value here and have it imported for use in the settings tab
+  const config = store.get();
+  const [currentConfig, setCurrentConfig] = useState(config);
```

Comment:
this is not needed, we can use the hook provided by the ConfigStore
`const config = store.useConfig()`

if you want a default value for showVerified you co do it like so 

`const showVerified = config?.showVerified ?? true;`

then you can also remove the useEffect on line 62

Reply:
using the store.useConfig() here seems to not work here

Reply:
had to rework it a little

Reply:
I just checked the code for the useConfig and it returns a function, a bit counterintuitive, so the usage would be:

```js
// outside the component
const useStoreConfig = store.useConfig();

// inside component
function ChartsList() {
  ...
  const store = useConfigStore()
  ...
```

Reply:
forgot to repost my comment I made in the sibling comment but I am using what we have elsewhere in the project for this implementation 
![image](https://github.com/user-attachments/assets/9c0dfdba-966e-4bd0-be6d-5bd740e2d878)
![image](https://github.com/user-attachments/assets/a9a16af8-1b6a-4464-b8e0-d0ad9c959f3a)